.github/workflows/main.yml

```diff
@@ -108,7 +108,14 @@ jobs:
       # Run Lighthouse audit (from any push)
       - name: Performance Audit with Lighthouse CI
         if: env.HAS_LHCI_GITHUB_APP_TOKEN == 'true' && github.event_name == 'push'
-        run: npm run performance
+        run: npm run performance -w angular-workspace
```

Comment:
Thiis branch still may be a work in progress but make sure to pull in the changes discussed in the comment thread as shown here: https://github.com/ni/nimble/compare/main...move-lh-to-angular-workspace

The goal is to not have component specific changes in the workflow file and I showed you how to accomplish that.

Reply:
I pulled the changes, but it's good that you highlighted this topic, for some reason I though that "npm run performance" will run only the tests from angular-workspace, that's why for making it more clear I added the workspace for both commands
![image](https://github.com/ni/nimble/assets/110180309/ea6e4a8a-1583-4dde-90f2-e21afd30669d)
but, as I tried just now, it was not needed